eng/pipelines/stress/http-linux.yml

```diff
@@ -0,0 +1,34 @@
+trigger: none
+
+pool:
+  name: Hosted Ubuntu 1604
+
+variables:
+  httpStressProject: src/System.Net.Http/tests/StressTests/HttpStress
+  sdkBaseImage: sdk-corefx-current
+  httpStressImage: httpstress
+
+steps:
+- checkout: self
+  clean: true
+  fetchDepth: 0
+  lfs: false
+
+- task: CmdLine@2
```

Comment:
There is also a short inline task syntax for cmdline just as for powershell.

```yml
- script: command....
  displayName: Name
```

And you can do multiline scripts as well:
https://github.com/dotnet/corefx/blob/b7e7c5584e84e3f086dc8a317c1a2f9cd999d956/eng/pipelines/linux.yml#L180-L187

Reply:
Ah wait, I changed this but I didn't push to the right fork